back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/import_project_ideas_service.rb

```diff
@@ -43,6 +43,8 @@ def generate_example_xlsx
           @project.allowed_input_topics.map { |t| t.title_multiloc[@locale] }.join '; '
         when 'number'
           5
+        when 'point'
+          rand(0.1..89.9).round(5)
```

Comment:
Does this output anything? Given that the fields that are output into the xlsx export are text fields, not point, this probably needs to do something different like target the field by suffix

Reply:
Yes, it outputs the values to the excel as shown in the screenshot in the PR description.